bioblend/galaxy/histories/__init__.py

```diff
@@ -37,6 +38,16 @@ def create_history(self, name=None):
             payload['name'] = name
         return self._post(payload)
 
+    def import_history(self, file_path=None, url=None, files_attached=False):
```

Comment:
Can you add a docstring here, something like
```suggestion
    def import_history(self, file_path=None, url=None):
        """
        Import a history from an archive on disk or a URL.
        
        :type file_path: str
        :param file_path: Path to exported history archive on disk.
        
        :type url: str
        :param url: URL for an exported history archive
        """
```
And drop the `files_attached` parameter, you can set this in the function body